src/Components/ArtistName.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from "prop-types"
+import "./Album.css"
+
+export const ArtistName = ({ artists }) => {
+  //Variable for more than 1 artists
+  const multipleArtists = artists.length > 1
+
+  //Map artist with the correct url
+  return (
+    <div className="artist-names">
+      {artists.map((artist, index) => (
+        <div className="artist-link" key={artist.id}>
+          <a
+            href={artist.external_urls.spotify}
+            target="_blank"
```

Comment:
What does this do? Is it to open a new tab when you press the link? 

/Wen & Martin